services/tables/src/main/java/com/linkedin/openhouse/tables/api/validator/impl/OpenHouseTablesApiValidator.java

```diff
@@ -141,6 +143,7 @@ private void validatePolicies(CreateUpdateTableRequestBody createUpdateTableRequ
                   "%s : %s",
                   policiesSpecValidator.getField(), policiesSpecValidator.getMessage())));
     }
+    replicationConfigValidator.validate(createUpdateTableRequestBody.getPolicies(), tableUri);
```

Comment:
Better to pass `replicationConfig` to validator here. `ReplicationConfigValidator` should only be concerned with `replicationConfig` unless there is more involved logic requiring the whole Policies?
Can do `policies!=null` check here.

Reply:
makes sense, updated to pass `Replication` through instead of `Policies`. since we can have multiple replication configs we can iterate through them in the validator